doc/examples/filters/plot_deconvolution.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+"""
+=====================
+Image Deconvolution
+=====================
+In this example, we deconvolve an image using Richardson-Lucy
+deconvolution algorithm.
+Richardson-Lucy deconvolution
+-----------------------------
+The algorithm is based on Bayesian-based iterative method
+where an image blurred by a point-spread function is 
+recovered. 
+.. [1] William Hadley Richardson, "Bayesian-Based Iterative 
+       Method of Image Restoration",
+       J. Opt. Soc. Am. A 27, 1593-1607 (1972)
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage import color, data, restoration
+
+from scipy.signal import convolve2d as conv2
+
+astro = color.rgb2gray(data.astro())
+psf = np.ones((5, 5)) / 25
+astro = conv2(astro, psf, 'same')
+camera += 0.1 * astro.std() * np.random.standard_normal(astro.shape)
+
+deconvolved = restoration.richardson_lucy(astro, psf, 5)
+
+fig, ax = plt.subplots(nrows=1, ncols=2, figsize=(8, 5),
+                       sharex=True, sharey=True,
+                       subplot_kw={'adjustable': 'box-forced'})
```

Comment:
What do you think about including the original image in the plot as well? I think it would help people to see where we are starting from. Then I would use the titles, "Original data", "Blurred", "Deconvolved using Richardson-Lucy".